content/04.results.md

```diff
@@ -14,15 +14,16 @@ Panel B shows the types and amounts of articles that we have used for analyses.
 
 ### Creation of an Annotated News Dataset
 
-We have analyzed the text and citations of 22,094 news-related articles hosted on "www.nature.com".
-The articles span 15 years from 2005 to 2020, and seven article types: Career Column, Career Feature, News, News and Views, News Feature, Technology Feature, and Toolbox.
+We have analyzed the text and citations of 16,080 news-related articles hosted on "www.nature.com".
+The articles span 15 years from 2005 to 2020, and five article types: Career Feature, News, News Feature, Technology Feature, and Toolbox.
+In our analyses, we focus primarily on journalist-written articles, but do include one analysis of scientist-written articles in the columns, Career Column and News and Views, which adds an additional 5,921 articles.
```

Comment:
```suggestion
We also include one analysis of the scientist-written news articles, Career Column and News and Views, as a comparison set of 5,921 articles.
```
I am not sure I love this wording, but something like this could help to intro the reader to the news types if they (like we) didn't get the distinctions initially.